graphql/splunktracing.go

```diff
@@ -0,0 +1,55 @@
+package graphql
+
+import (
+	"context"
+
+	"github.com/99designs/gqlgen/graphql"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+)
+
+type SplunkTracing struct{}
+
+func (SplunkTracing) ExtensionName() string {
+	return "SplunkTracing"
+}
+
+func (SplunkTracing) Validate(graphql.ExecutableSchema) error {
+	return nil
+}
+
+func (SplunkTracing) InterceptResponse(ctx context.Context, next graphql.ResponseHandler) *graphql.Response {
+	rc := graphql.GetOperationContext(ctx)
+	if rc == nil {
+		// There was an invalid operation context, so we can't do anything
+		grip.Critical(message.Fields{
+			"message": "no operation context found",
```

Comment:
Can you make this message more explicit, specifically that this is the operation context for a graphql request?